moveit_ros/planning_interface/move_group_interface/src/wrap_python_move_group.cpp

```diff
@@ -235,19 +235,19 @@ class MoveGroupInterfaceWrapper : protected py_bindings_tools::ROScppInitializer
     convertListToPose(pose, msg.pose);
     msg.header.frame_id = getPoseReferenceFrame();
     msg.header.stamp = ros::Time::now();
-    return place(object_name, msg);
+    return place(object_name, msg) == MoveItErrorCode::SUCCESS;
```

Comment:
Hm, does it make sense to add a "bool ok()" method to simplify these lines?

Reply:
I decided not to add it in the end because right now it still "feels" like comparing plain old return values and I didn't want to change that impression to "we have a fancy status object", but I'm fine with either one

Another argument for having an "bool ok()" function is that there could in theory be several positive responses ("found some path" vs "found optimal path" or something like that)

I would name it "bool succeeded()" as in 

     return place(object_name, msg).succeeded();

Reply:
i personally prefer ``place(object_name, msg) == MoveItErrorCode::SUCCESS;`` because the currently proposed API is not very readable and a new user will have to dig deep into the code to understand that ``.suceeded()`` doesn't actually call any functionality. ``suceeded()`` is syntactic sugar that obfuscates the already easy to understand `` == MoveItErrorCode::SUCCESS;``